.buildkite/pipeline.release.yml

```diff
@@ -0,0 +1,148 @@
+---
+env:
+  BUILD_ENVIRONMENT_IMAGE: "docker.cloudsmith.io/grapl/raw/pulumi-build-env:3.26.1"
+  BUILDKITE_PLUGIN_VAULT_ENV_SECRET_PREFIX: "secret/data/buildkite/env"
+
+steps:
+
+  # TODO: Add testing-only logic to this pipeline
+  # - only build the provider binaries; don't actually release
+  # - push SDKs to Cloudsmith, rather than PyPI / NPM
+  #
+  # We would still be creating a tag, which is unfortunate, but
+  # currently unavoidable.
+
+  - block: ":writing_hand: Release Information"
+    prompt: |
+      Please specify the following release parameters:
+    fields:
+      - text: Release Tag (start with a "v", please)
+        key: release-tag
+      # TODO: add options for pre-release, etc.?
+
+      # TODO: Add intelligence for automatically creating an
+      # appropriate release tag, based on what's already in the
+      # repository. If we just say "it's a new major/minor/patch
+      # release" here, then we can use that to compute the final tag
+      # elsewhere in code. That may be less for people to juggle in
+      # their heads.
+
+  - label: ":github: Create Release Tag"
+    command:
+      .buildkite/scripts/git-tag
+    env:
+      BUILDKITE_CLEAN_CHECKOUT: true
+    plugins:
+      - grapl-security/vault-login#v0.1.2
+      - grapl-security/vault-env#v0.1.0:
+          secrets:
+            - GITHUB_TOKEN
+    agents:
+      queue: github-release
+
+  - wait
+
+  - group: "Build SDKs"
+    steps:
+      - label: ":python: Build Python SDK"
+        key: build-python-sdk
+        command:
+          - make python-sdk
+        plugins:
+          - improbable-eng/metahook#v0.4.1:
+              # We need to ensure we have the tag we just created
+              pre-command: git fetch --unshallow --tags
+          - docker#v3.11.0:
+              image: "${BUILD_ENVIRONMENT_IMAGE}"
+        artifact_paths:
+          - sdk/python/bin/dist/*.tar.gz
+
+      - label: ":nodejs: Build NodeJS SDK"
+        key: build-nodejs-sdk
+        command:
+          - make nodejs-sdk
+        plugins:
+          - improbable-eng/metahook#v0.4.1:
+              # We need to ensure we have the tag we just created
+              pre-command: git fetch --unshallow --tags
+          - docker#v3.11.0:
+              image: "${BUILD_ENVIRONMENT_IMAGE}"
+        artifact_paths:
+          - sdk/nodejs/bin/dist/*.tgz
+
+  - wait
+
+  # This will build the providers, package them up, upload them to
+  # Github, and announce them.
+  #
+  # We're intentionally publishing the SDKs after this; see below.
+  - label: ":github: Release!"
+    command:
+      # TODO: --skip-announce is only present for testing
+      # TODO: we really want this queue to be bigger so we can build more quickly
+      # TODO: I don't really love the fact that we're having to build these artifacts _in the release pipeline_.
+      - goreleaser release --skip-announce
```

Comment:
should skip-announce be removed, then?
i just need a 'lifetime in these TODOs

Reply:
Yup, I've removed it now.